packages/core/src/index.ts

```diff
@@ -54,7 +54,13 @@ export { createTransport } from './transports/base';
 export { makeOfflineTransport } from './transports/offline';
 export { makeMultiplexedTransport } from './transports/multiplexed';
 export { SDK_VERSION } from './version';
-export { getIntegrationsToSetup, addIntegration } from './integration';
+export {
+  getIntegrationsToSetup,
+  addIntegration,
+  // eslint-disable-next-line deprecation/deprecation
```

Comment:
If we are using deprecated functions, I think we should disable this rule instead of overriding it everywhere

Reply:
The idea here was to make it easier to spot that this should be removed in v8 🤔 no strong feelings, I just wanted to make it clear that this is a temporary thing that will be removed in v8 again - generally, we should avoid using deprecated stuff ourselves, so I think the rule is good.